test/verifiedTests/analysis/testMultiSpeciesModelling/testCreatePanModels.m

```diff
@@ -26,7 +26,8 @@
 % create the pan-models on species level
 panPath=[pwd filesep 'panSpeciesModels'];
 
-createPanModels(modPath,panPath,'Species','AGORA',numWorkers);
+builtTaxa = {'Acinetobacter junii','Bacteroides ovatus','Enterobacter cloacae','Megasphaera elsdenii','Ruminococcus flavefaciens'};
+createPanModels(modPath,panPath,'Species','AGORA_infoFile.xlsx',numWorkers,builtTaxa)
```

Comment:
Hi @almut-heinken 
Is it possible that we move these models to COBRA.models and load them using "getDistributedModel" to avoid downloading them? or are these models updating regularly and it is important to always download their latest version?

Reply:
Hi @farid-zare 
I modified the code to rely on a subset of models. I also opened a pull request on COBRA.models to upload the neccessary models.

Reply:
Thanks a million @almut-heinken. This is great.